app/models/corpus_item_model.py

```diff
@@ -4,3 +4,4 @@
 class CorpusItemModel(BaseModel):
     id: str
     topic: str = None
+    scheduled_date: str = None  # TODO: Query this value from scheduledSurface.items [DIS-452]
```

Comment:
Same here, this is specific to new tab, we don't schedule anything for home or similar content recs, so this field doesn't make any sense for them.  I assume those different recommendation areas will grow in different directions even more over time. Why don't we create different minimal classes to support new tab.

Reply:
Good catch. `scheduled_date` is a property of [ScheduledSurfaceItem](https://studio.apollographql.com/graph/pocket-client-api/schema/reference/scalars/Date?query=scheduled&variant=current), not CorpusItem.